src/control/lib/spdk/src/nvme_control.c

```diff
@@ -265,15 +262,15 @@ nvme_wipe_namespaces(void)
 		snprintf(ret->info, sizeof(ret->info),
 			 "spdk_nvme_probe() (%d)\n", rc);
```

Comment:
Same note here, now that the rc is returned in ret

Reply:
correct